core/interfaces/i_legacy_procedure_blocks.ts

```diff
@@ -28,7 +28,7 @@ export interface LegacyProcedureDefBlock {
 
 /** @internal */
 export function isLegacyProcedureDefBlock(
-  block: Object,
+  block: object,
```

Comment:
Probably too late to change this now, but perhaps this should have been `block: Block`…

Reply:
I don't think this particularly matters since it's legacy.